spec/plainyearmonth.html

```diff
@@ -27,9 +27,12 @@ <h1>Temporal.PlainYearMonth ( _isoYear_, _isoMonth_ [ , _calendarLike_ [ , _refe
         1. If _referenceISODay_ is *undefined*, then
           1. Set _referenceISODay_ to *1*<sub>𝔽</sub>.
         1. Let _y_ be ? ToIntegerOrInfinity(_isoYear_).
+        1. If _y_ is +∞ or -∞, throw a *RangeError* exception.
```

Comment:
Extreme nitpick: Unicode minus sign
```suggestion
        1. If _y_ is +∞ or −∞, throw a *RangeError* exception.
```
(feel free to disregard, we can always fix these up later)

Reply:
i'm pretty sure we use a normal en dash for negation in the spec